Tests/StoreKitUnitTests/TestHelpers/AvailabilityChecks.swift

```diff
@@ -55,6 +55,12 @@ enum AvailabilityChecks {
         }
     }
 
+    static func iOS16_4APIAvailableOrSkipTest() throws {
```

Comment:
Probably don't need it since I think we can use 16.0 for everything (other than `billingIssue`)